src/fa/game_session.py

```diff
@@ -177,13 +178,41 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.full.log', 'a+') as dst: dst.write(src.read())
 
     def _exited(self, status):
         self._game_connection = None
         self.state = GameSessionState.OFF
         self._logger.info("Game has exited with status code: {}".format(status))
         self.send('GameState', ['Ended'])
 
+        replayID=''
+        with open(util.LOG_FILE_GAME,'r') as inF:
+            for line in inF:
+                if 'gpgnet://localhost/' in line:
+                    try:
+                        replayID=line.split('localhost/')[1].split('/')[0]
+                    except:
+                        replayID='ERR'
+        try:
+            nb_replayid_files=0
+            list_game_replayid_files=os.listdir(util.LOG_DIR)
+            size_replayid_files=len(list_game_replayid_files)-1
+            while size_replayid_files >= 0:
+                if "Replayid" not in list_game_replayid_files[size_replayid_files]:
+                    list_game_replayid_files.remove(list_game_replayid_files[size_replayid_files])
+                else:
+                    nb_replayid_files+=1
+                size_replayid_files-=1
+            if nb_replayid_files >=11:
+                os.remove(util.LOG_DIR+"\\"+list_game_replayid_files[0])
+            with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.Replayid'+str(replayID)+'.log', 'w+') as dst: dst.write(src.read())
+            with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.full.log', 'a+') as dst: dst.write(src.read())
+            with open(util.LOG_FILE_GAME, 'w') as src : pass
+        except :
+            with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.full.log', 'a+') as dst: dst.write(src.read())
```

Comment:
Duplicated above line. Maybe let's just do it unconditionally outside of try/catch?